docs/source/json-api/search-query-language.rst

```diff
@@ -16,7 +16,7 @@ Fallback rule
 
 Unless otherwise required by one of the other rules below or to follow,
 values are interpreted according to :doc:`lf-value-specification`, and
-compared for equality.
+compared for equality, unless they are lists or maps.
```

Comment:
Can we please make it more clear that lists and maps are not allowed in queries.

Reply:
How about c63f0917ef ?